src/Files.App/Views/Layouts/DetailsLayoutPage.xaml

```diff
@@ -964,6 +964,7 @@
 											x:Name="SelectionCheckbox"
 											Width="20"
 											MinWidth="0"
+											MinHeight="0"
```

Comment:
Before:
![image](https://github.com/files-community/Files/assets/62196528/0f459740-0111-4282-a794-1af18dc60a06)

After:
![image](https://github.com/files-community/Files/assets/62196528/47ee86aa-c06e-4951-ae57-3b3ab0597155)

Reply:
The before looks better

Reply:
I'll revert. But I think we could add 2px or 2px border in the future.

Reply:
Reverted